src/components/ArtisanSwitch/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import cls from 'classnames';
+import { Switch } from '@headlessui/react';
+import styles from './styles.module.css';
+
+import {
+  flex,
+  padding,
+  borders,
+  colors,
+  layout,
+} from '../../theme';
+
+export default function ArtisanSwitch({ switchLabel, setToggle }) {
+  const [enabled, setEnabled] = useState(false);
+
+  const changeHandler = (val) => {
+    setEnabled(val);
+    setToggle(val);
+  };
+
+  const switchClass = cls(
+    styles.switch,
+    colors.bgSlate900,
+    { [styles.enabled]: enabled },
+    { [styles.disabled]: !enabled },
```

Comment:
You can actually just combine these into a single object.

```suggestion
    { 
      [styles.enabled]: enabled,
      [styles.disabled]: !enabled 
    },
```

Reply:
Leaving this comment here, but it applies to everywhere where you're doing separate objects, (like line 32-33)